packages/react-scripts/config/webpack.config.js

```diff
@@ -192,21 +192,21 @@ module.exports = function (webpackEnv) {
     if (preProcessor) {
       loaders.push(
         ...[
-          {
-            loader: require.resolve('resolve-url-loader'),
-            options: {
-              sourceMap: isEnvProduction ? shouldUseSourceMap : isEnvDevelopment,
-              root: paths.appSrc,
-            },
-          },
-          // Because sass-loader is the most expensive, so put cache-loader here to only cache sass-loader
+          // Moving cache-loader above resolve-url-loader is because its cache files are small and it is faster to read the cache files, and it saves more time than cache-loader below resolve-url-loader, we can still speed up the process of compiling sass to css since sass-loader is the most expensive
```

Comment:
We may not need this comment in the code to explain why we have moved this here. Could you put it in the PR instead just for history reference?

Given this is something we have added to BRS we don't need to explain this here, this comment is more a PR comment vs a code comment